start-site/src/main/java/io/spring/start/site/extension/dependency/activemq/ActiveMQProjectGenerationConfiguration.java

```diff
@@ -45,10 +47,26 @@ ServiceConnectionsCustomizer activeMQServiceConnectionsCustomizer(DockerServiceR
 
 	@Bean
 	@ConditionalOnPlatformVersion("3.2.0-M1")
```

Comment:
Shouldn't that read `@ConditionalOnPlatformVersion("3.3.0-M2")`?

Reply:
Yes, good catch! I have updated the PR.